nbs/tutorials/extensions.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    _LAYOUT_STR = Template("::: {$layout}\n${content}\n")
</pre><p>Is it intentional that the <code>$</code> is inside the braces in one case and outside in the other? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Yep! Because quarto wants it as `::: {.column-margin}` so I wrap whatever `.something` in the `{}`